### PR TITLE
Fixing bugs appearing during environment setup

### PR DESCRIPTION
**Pull request by luckysori**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -156,13 +156,19 @@ services:
...
 
   adminer:
     container_name: adminer
     image: adminer
     restart: always
     ports:
       - 8888:8080
+    networks:
+      vtto:
+        ipv4_address: 10.5.0.9
```
Comment (bonomat) [inline docker-compose.yml:171]: If noone is using this container we can actually remove it. I've only used it during the initial work at the db :)

**Review by bonomat — APPROVED** [review]
Thanks for the fixes

File: docker-compose.yml (hunk 1 above)
Comment (luckysori) [reply docker-compose.yml:171]: Are you referring to the `adminer`? I'll remove it if it's not being used at all.

Comment (bonomat) [reply docker-compose.yml:171]: yes

Comment (luckysori) [reply docker-compose.yml]: Done: https://github.com/get10101/10101/pull/285/commits/bc64ddc82ebdbb9a0ce8365a32078cfbb36066ec.

File: mobile/native/src/ln_dlc/mod.rs [hunk 2]
```diff
@@ -116,9 +116,18 @@ pub fn run(data_dir: String) -> Result<()> {
         let node = Arc::new(Node { inner: node });
 
         // todo: should the library really be responsible for managing the task?
```
Comment (holzeis) [inline mobile/native/src/ln_dlc/mod.rs:118]: nit: I guess it should not, but then we can remove the comment 🙂

**Pull request merged**